tests/test_server.py

```diff
@@ -584,6 +584,15 @@ def pong(*values):
         print("pong called")
         osc_2.answer(b'/ping', [True])
 
+    osc_3 = OSCThreadServer()
+    osc_3.listen(default=True, port=8000)
```

Comment:
This can cause tests to fail if the port is used by something else, i'd rather let the OS assign the port like the others, and use `port=osc_3.getaddress()[1]` in the answer.